client/src/util.rs

```diff
@@ -237,7 +237,8 @@ pub struct Api<'a> {
 impl<'a> Api<'a> {
     pub fn new(server: &'a ServerInfo) -> Self {
         let agent = AgentBuilder::new()
-            .timeout(Duration::from_secs(15))
+            // Some platforms (e.g. OpenBSD) can take longer to complete the first WireGuard handshake, hence a lower timeout value could result in an unwarranted failure
```

Comment:
Formatting nit: please keep the lines to ~100 chars max. (stable rustfmt doesn't format comments yet unfortunately)